src/app/shared/table/filters/date-range-filter.ts

```diff
@@ -0,0 +1,27 @@
+import { FilterType, TableFilterDef } from 'app/types/Table';
```

Comment:
Rename date-range-filter.ts -> date-range-table-filter.ts
Follow what is already in place.
All filter class names should end with *table-filter.ts